steering/elections/2022/election.yaml

```diff
@@ -0,0 +1,16 @@
+name: 2022 Steering Committee Election
+organization: Istio
+start_datetime: 2022-10-03 00:00:01
+end_datetime: 2022-10-16 11:59:59
+no_winners: 4
+allow_no_opinion: True
+delete_after: True
+show_candidate_fields:
+  - employer
+  - slack
+election_officers:
+  - rvennam
+  - cetezadi
+eligibility: A [project member](https://github.com/istio/community/blob/master/ROLES.md#member) who has had at least one Pull Request merged in the past 12 months. See [the election guide](https://github.com/istio/community/tree/master/steering/elections/2022)
```

Comment:
I don't think markdown gets evaluated in this field.

Reply:
https://github.com/elekto-io/elekto.meta.test/blob/main/elections/2021/TOC/election.yaml says it does